plugin.video.manoto/default.py

```diff
@@ -34,29 +34,29 @@ def getStreamsFromPlayList(playlist):
         except urllib2.HTTPError, he:
         	print("HTTP error trying to open playlist")
```

Comment:
Please remove the print statements from your modules. Please use xbmc.log or a support routine instead.